### PR TITLE
add safety guard to the sequence

### PR DESCRIPTION
**Pull request by libotony**
# Description

A safety precaution for a successor who is willing to get hands dirty to the sequence.go file.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] New and existing E2E tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have not added any vulnerable dependencies to my code

### CODE REVIEW

File: logdb/sequence.go [hunk 1]
```diff
@@ -23,6 +23,12 @@ const (
 	logIndexMask = (1 << logIndexBits) - 1
 )
 
+func init() {
+	if blockNumBits+txIndexBits+logIndexBits > 63 {
+		panic("total bits in sequence exceeds max bit allowed")
+	}
+}
```
Comment (otherview) [inline logdb/sequence.go:30]: Given these are consts, should this be in a test ?

**Review by otherview — APPROVED** [review]

File: logdb/sequence.go (hunk 1 above)
Comment (libotony) [reply logdb/sequence.go:30]: I don't have a strong opinion on this, it is for someone edited the bits then started from source. Adding to tests will make it clean?

Comment (otherview) [reply logdb/sequence.go:30]: I guess my thought process is that if someone modifies the code and doesn't run the tests, it's likely that they might be setting themselves up for bugs.

Comment (leszek-vechain) [reply logdb/sequence.go:30]: what Pedro said :)

Comment (libotony) [reply logdb/sequence.go]: Updated!

**Review by otherview — APPROVED** [review]

**Pull request merged**